src/pipelines.py

```diff
@@ -0,0 +1,50 @@
+"""
+    Pipelines for cyberowl scraper.
+"""
+from mdtemplate import Template
+
+
+class AlertPipeline:
+    """
+    AlertPipeline class
+    """
+
+    result: list = []
+
+    def remove_special_characters(self, text):
+        """
+        Remove special characters from text.
+        """
+        return (
+            text.replace("\n", "")
```

Comment:
not sure how far this list can grow! but I think u should create list and loop over it to remove whatever you want
e.g. 
```python
special_characters = ['\n','\r','  ','|']
return text.translate({ord(charachter): '' for charachter in special_characters})
```

Reply:
Yesss!! Great suggestion!